src/nix/profile-remove.md

```diff
@@ -11,7 +11,7 @@ R""(
 * Remove all packages:
 
   ```console
-  # nix profile remove '.*'
+  # nix profile remove --regex '.*'
```

Comment:
Why not `--all`?

Might make sense to add an example of removing all packages that match some regex (maybe python is a good example).